roboflow/core/version.py

```diff
@@ -587,17 +589,20 @@ def deploy(self, model_type: str, model_path: str) -> None:
                     if file in ["model_artifacts.json", "state_dict.pt"]:
                         raise (
                             ValueError(
-                                f"File {file} not found. Please make sure to provide a valid model path."
+                                f"File {file} not found. Please make sure to provide a"
+                                " valid model path."
                             )
                         )
 
         res = requests.get(
-            f"{API_URL}/{self.workspace}/{self.project}/{self.version}/uploadModel?api_key={self.__api_key}&modelType={model_type}&nocache=true"
+            f"{API_URL}/{self.workspace}/{self.project}/{self.version}"
```

Comment:
Is this valid?

Reply:
Yes it is (minimal example for you) 

![image](https://github.com/roboflow/roboflow-python/assets/1688848/4d9c7b9a-1e1c-4c52-9a2e-8506c0bfb3db)